lsp-ui-imenu.el

```diff
@@ -331,7 +338,25 @@ Return the updated COLOR-INDEX."
 (define-derived-mode lsp-ui-imenu-mode special-mode "lsp-ui-imenu"
   "Mode showing imenu entries.")
 
-(defun lsp-ui-imenu-enable (_enable))
+(defun lsp-ui-imenu--refresh ()
```

Comment:
I wonder if we should make this function a command and bind it to "r" in `lsp-ui-imenu-mode`.

Reply:
Added!